### PR TITLE
Version/7.0.0

### PR DESCRIPTION
**Pull request by mejango**
- redeployed contracts following folder restructure.
- verified newly deployed contracts.
- updated postmortem with details about the new deployment and latest planned recourse.

### CODE REVIEW

File: deploy/2.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
+const { ethers } = require('hardhat');
+
+/**
+ * Deploys a second version of many contracts for projects to migrate onto as a conequence of https://github.com/jbx-protocol/juice-contracts-v2/pull/268.
```
Comment (drgorillamd) [inline deploy/2.js:4]: tyop> consequence

Comment (drgorillamd) [reply deploy/2.js:4]: sh*t, 'typo'

File: security/postmortem/5.24.2022.md [hunk 2]
```diff
@@ -90,6 +106,9 @@ Projects can re-distribute tokens to their holders as follows:
 
 If a project's treasury funds are greater than its current distribution limit, they can remove funds out of the current payment terminal by reconfiguring their next funding cycle to raise the distribution limit so that it encompasses all of the treasury's funds, routing them to an address that will facilitate their transfer to the updated payment terminal via a call to `addToBalance(...)`.
 
+### Update May 28, 2022
+
+Given the relatively few projects that and project token holders that are in need of taking action to mitigate risks due to the bug, JuiceboxDAO contributors are encouraging projects to re-issue their ERC-20s, and will recreate each project's activity and token distribution with its own funds. Projects will no longer need to concern themselves with re-distributing tokens.
```
Comment (drgorillamd) [inline security/postmortem/5.24.2022.md:111]:
```suggestion
Given the relatively few projects and project token holders that are in need of taking action to mitigate risks due to the bug, JuiceboxDAO contributors are encouraging projects to re-issue their ERC-20s, and will recreate each project's activity and token distribution with its own funds. Projects will no longer need to concern themselves with re-distributing tokens.
```

**Review by drgorillamd — APPROVED** [review]

File: security/postmortem/5.24.2022.md (hunk 2 above)
Comment (mejango) [reply security/postmortem/5.24.2022.md:111]: nice

**Pull request merged**